code_maintenance/upload_vendor_asset.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Uploads the vendored dependency tarball as a release asset.
+"""
+
+import os
+import sys
+
+from github import Github
+
+
+def main():
+    """
+    Main function
+    """
+    api_key = os.environ.get("GITHUB_API_KEY")
+    if api_key is None:
+        raise RuntimeError("GITHUB_API_KEY environment variable is required")
```

Comment:
What kind of key is this? The reason I'm unsure is that when I was trying out the API I authenticated as me. Would this be a GitHub Secret instead?

Reply:
This is a personal access token.